adapters/mobkoi/params_test.go

```diff
@@ -0,0 +1,63 @@
+package mobkoi
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/prebid/prebid-server/v3/openrtb_ext"
+)
+
+func TestValidParams(test *testing.T) {
+	validator, err := openrtb_ext.NewBidderParamsValidator("../../static/bidder-params")
+
+	if err != nil {
+		test.Fatalf("Failed to fetch the json-schemas. %v", err)
+	}
+
+	for _, validParam := range validParams {
+		err := validator.Validate(openrtb_ext.BidderMobkoi, json.RawMessage(validParam))
+
+		if err != nil {
+			test.Errorf("Schema rejected Mobkoi params: %s\nError: %v", validParam, err)
+		}
+	}
+}
+
+func TestInvalidParams(test *testing.T) {
+	validator, err := openrtb_ext.NewBidderParamsValidator("../../static/bidder-params")
+
+	if err != nil {
+		test.Fatalf("Failed to fetch the json-schemas. %v", err)
+	}
+
+	for _, invalidParam := range invalidParams {
+		err := validator.Validate(openrtb_ext.BidderMobkoi, json.RawMessage(invalidParam))
+
+		if err == nil {
+			test.Errorf("Schema allowed unexpected params: %s", invalidParam)
+		}
+	}
+}
+
+var validParams = []string{
+	`{"placementId":"abc"}`,
+	`{"placementId":"abc", "adServerBaseUrl":"https://adserver.mobkoi.com"}`,
+	`{"adServerBaseUrl":"http://dev.mobkoi.com"}`,
+	`{"placementId":"abc", "adServerBaseUrl":"https://adserver.mobkoi.com"}`,
+}
+
+var invalidParams = []string{
+	``,
+	`null`,
+	`true`,
+	`1`,
+	`1.0`,
+	`[]`,
+	`{}`,
```

Comment:
After this last commit, test cases are failing. I ran `validate.sh` locally and got:
```
--- FAIL: TestInvalidParams (0.14s)
    params_test.go:37: Schema allowed unexpected params: {}
    params_test.go:37: Schema allowed unexpected params: {"placement_id":"abc"}
FAIL
```
After making `placementId` non-required in `static/bidder-params/mobkoi.json`, an empty `imp[i].ext` now pases param validation and this entry, `{}`, should be moved to what we expect to be `validParams`